<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Puertos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="portNameCB"/>
      </item>
      <item>
       <widget class="QLabel" name="portNamelabel">
        <property name="text">
         <string>Baudios</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="baudRateCB"/>
      </item>
      <item>
       <widget class="QLabel" name="description">
        <property name="text">
         <string>Descripci√≥n</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="baudRate">
        <property name="text">
         <string>Baudios</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="serialNumber">
        <property name="text">
         <string>SerialNumber</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Parar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cleanButton">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelTiempo">
          <property name="text">
           <string>Tiempo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="tiempo">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Seg.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Graficar cada</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignVCenter">
         <widget class="QSpinBox" name="frecgraph">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Datos</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Serial Write</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="serialDataLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="writeSerial">
          <property name="text">
           <string>WriteSerial</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="styleSheet">
        <string notr="true">
QPushButton{
color : white;
background-color: #5cb85c;
border: 2px solid #4cae4c;
border-radius: 3px;
padding: 5px;
min-width: 80px;
}

QPushButton:hover {
background-color: #47a447;
border: 2px solid #398439;
}

QPushButton:pressed {
border: 3px solid #398439;
}</string>
       </property>
       <property name="text">
        <string>Conectar</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExit">
       <property name="styleSheet">
        <string notr="true">QPushButton{
color : white;
background-color: rgb(255, 0, 0);
border: 2px solid;
border-radius: 3px;
padding: 5px;
min-width: 80px;
}

QPushButton:hover {
background-color: rgb(170, 0, 0);
border: 2px solid ;
}

QPushButton:pressed {
border: 3px solid rgb(102, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
